scripts/build.py

```diff
@@ -202,7 +209,7 @@ def write_to_file_stream(file_stream, content):
         file_stream: file. A stream handling object to do write operation on.
         content: str. String content to write to file object.
     """
-    file_stream.write(content)
+    file_stream.write(utils.convert_to_unicode(content))
```

Comment:
What's the difference between this and `python_util`'s version?

Reply:
This is an existing function that I used. It converts a string to unicode whereas the one in python_utils converts to python2's str.